ReactWindows/ReactNative.Net46.Tests/Modules/NetInfo/NetInfoModuleTests.cs

```diff
@@ -34,8 +33,11 @@ public void NetInfoModule_JsonResponse()
         }
 
         [Test]
+        [Apartment(ApartmentState.STA)]
         public void NetInfoModule_Event()
         {
+            SetDispatcherForTest();
+
```

Comment:
The NetInfo tests themselves are calling OnResume, which will assert on the dispatcher availability... Leaving code that sets the dispatcher ahead of the test execution...